extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/usecase/model/ConsumptionInputDto.java

```diff
@@ -0,0 +1,41 @@
+package de.sovity.edc.ext.wrapper.api.usecase.model;
+
+import de.sovity.edc.ext.wrapper.api.common.model.PolicyDto;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.Map;
+
+/**
+ * Input for a consumption process.
+ *
+ * @author Ronja Quensel
+ */
```

Comment:
retire java doc in favor of swagger annotations for documenation of functionality